### DISCUSSION TITLE
v4.3.0 release

### DISCUSSION THREAD

**Discussion by Adam-it** [pnp/vscode-viva#381] [Announcements]
## ⭐ What's Changed

👉 Added remove confirmation prompt when removing SPFx solution from app catalog
👉 Added `/manage` GitHub Chat command
👉 Added generating code tour for upgrade and validate SPFx solution actions
👉 Added upgrade SPFx project action
👉 Added SPFx Fast Serve optional dependency to the scaffolding form
👉 Improved sign in experience with caching of Client ID and Tenant ID

## 💪 Contributors

* @Saurabh7019 - Contributor of the month 🏆
* @nicodecleyre  - Code tour expert 👨‍🎓
* @Adam-it - I think I did something here 🤷‍♂️

Thank you for your awesome work in improving the product. You all rock 🤩

📃 Bog post 👉: https://pnp.github.io/blog/post/spfx-toolkit-vscode-v-4-3-release/

**Full Changelog**: https://github.com/pnp/vscode-viva/compare/v4.2.0...v4.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/pnp/vscode-viva/releases/tag/v4.3.0'>v4.3.0 release</a>.</em>

**Comment by nicodecleyre** [comment]
Woot- woot congratulations @Saurabh7019 with contributor of the month 🤩🤩
